LeaderboardBackend.Test/TestApi/TestApiFactory.cs

```diff
@@ -51,21 +53,13 @@ public TestApiClient CreateTestApiClient()
         return new TestApiClient(client);
     }
 
-    public void InitializeDatabase()
+    public static void InitializeDatabase(ApplicationContext dbContext)
```

Comment:
Why pass the context as an argument to initialising and resetting the DB? Failing to see why we can't resolve the context within this file, like we currently do.